exporter/kafkaexporter/config.go

```diff
@@ -77,6 +77,21 @@ type Producer struct {
 	//   1 -> WaitForLocal. waits for only the local commit to succeed before responding ( default )
 	//   -1 -> WaitForAll. waits for all in-sync replicas to commit before responding.
 	RequiredAcks sarama.RequiredAcks `mapstructure:"required_acks"`
+
+	// Compression Codec used to produce messages
+	// https://pkg.go.dev/github.com/Shopify/sarama@v1.30.0#CompressionCodec
+	// The options are:
+	//   0 -> None
```

Comment:
Instead of directly using the numeric constants, how about accepting the strings here to make it easier to config for users?

Reply:
I could do that. Just did it this way to match the way RequiredAcks is done above

Reply:
We probably want to cleanup RequiredAcks in the future, but I think for this PR it's fine to diverge for this setting, it'll make configs for compression much more readable